cmd/commands/runtime.go

```diff
@@ -274,7 +279,7 @@ func RunRuntimeList() error {
 	for _, rt := range runtimes {
 		status := "N/A"
 		namespace := "N/A"
-		cluster := "N/A"
+		cluster := rt.Cluster
 		name := "N/A"
 		version := "N/A"
```

Comment:
version?

Reply:
resolved. Along with namespace and name, which are also not null